packages/features/bookings/Booker/components/LargeCalendar.tsx

```diff
@@ -39,14 +44,32 @@ export const LargeCalendar = ({ extraDays }: { extraDays: number }) => {
     .add(extraDays - 1, "day")
     .toDate();
 
+  // HACK: force rerender when overlay events change
+  // Sine we dont use react router here we need to force rerender (ATOM SUPPORT)
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  useEffect(() => {}, [displayOverlay]);
```

Comment:
Idk a better way to do this since queryParams in booker dont seem to auto re-render 🤔

Reply:
i tried using  

```
import { useSearchParams } from "next/navigation";
  const searchParams = useSearchParams();
  const displayOverlay = searchParams.get("overlayCalendar") === "true";
```

and does re render when any param is changed even when i remove this useEffect

Reply:
As mentioned in DMs we try not to use nextjs stuff in booker due to it being an atom we want to support globally in react <3